docs/libertyDev.md

```diff
@@ -101,12 +101,24 @@ The following is a technology preview. The features and parameters described bel
 
 Start a Liberty server in a local container using the Dockerfile that you provide. An alternative to the `libertyDevc` task is to specify the `libertyDev` task with the `--container` option. 
 
-The Dockerfile must copy the application .war file and the server configuration files that the application requires into the container. A sample Dockerfile is shown in [Building an application image](https://github.com/openliberty/ci.docker/#building-an-application-image). Note that the context of the `docker build` command used to generate the container image is the directory containing the Dockerfile. When dev mode exits the container is stopped and deleted and the logs are preserved as described below.
-
 You need to install the Docker runtime (e.g. Docker Desktop) locally to use this Gradle task. You can examine the commands used to build and run the container by viewing the console output of dev mode.
 
 When dev mode runs with container support it still provides the same features. It monitors files for changes and runs tests either automatically or on demand. This mode also allows you to attach a debugger to work on your application. You can review the logs generated by your server in the Liberty directory in your project e.g. build/wlp/usr/servers/defaultServer/logs.
 
+The Dockerfile must copy the application .war file and the server configuration files that the application requires into the container. A sample Dockerfile is shown in [Building an application image](https://github.com/openliberty/ci.docker/#building-an-application-image). Note that the context of the `docker build` command used to generate the container image is the directory containing the Dockerfile. When dev mode exits the container is stopped and deleted and the logs are preserved as described.
+
+Dev mode offers different levels of file tracking and deployment depending on the way the file is specified in the Dockerfile. 
+1. When you use the COPY command on an individual file dev mode can track file changes and hot deploy them to the container subject to the limitations below. **This is the most productive way to deploy files.**
+- E.g. `COPY src/main/liberty/config/server.xml /config/` 
+- Note that the Dockerfile must copy only one .war file for the application.  Other application archive formats or multiple .war files are not supported.
+2. You can use the COPY command to deploy an entire directory and its sub-directories. In this case, dev mode will detect file changes and use a regular deployment strategy. Dev mode will rebuild the image and restart the container. 
+3. The ADD command can be used on individual files including tar files as well as on directories. Again, dev mode will rebuild the image and restart the container. 
```

Comment:
I suggest mentioning that this only happens on file changes, e.g. "Again, dev mode will rebuild the image and restart the container upon file changes. " or "when it detects file changes"

Reply:
/nod